api/hooks/discordBot/commands/sdtd/status.js

```diff
@@ -53,17 +53,27 @@ class Status extends Commando.Command {
       onlinePlayersStringList += `${he.decode(player.name)}, `;
     });
 
+    let bloodMoonDay;
 
-    let nextHorde = (Math.trunc(serverInfo.stats.gametime.days / 7) + 1) * 7;
-    const daysUntilHorde = nextHorde - serverInfo.stats.gametime.days;
+    try{
+      bloodMoonDay = await sails.helpers.sdtdApi.executeConsoleCommand(
+        SdtdServer.getAPIConfig(sdtdServer),
+        `ggs BloodMoonDay`
+      );
+    } catch (error) {
+      sails.log.warn(`Hook - discordBot:status - ${error}`);
+      sails.log.error(error);
+    }
 
+    let nextHorde = parseInt(bloodMoonDay.result.split(` = `)[1]);
```

Comment:
so the slight issue here is if get bloodmoon days fails, it goes into the catch, reports data, then tries to go use that data.

So I suggest defining nextHorde before the try as the default value `(Math.trunc(serverInfo.stats.gametime.days / 7) + 1) * 7` thenn inside the try, override nextHorde with the parseInt stuff

Reply:
What about something like 
```
if(bloodMoonDay) {
    //do the math for days until BM, and set up message for embed
} else {
    //error message where days until BM is about being unable to retrieve
}```

I think that would be a better solution than showing a potentially incorrect value

Reply:
That's fine, instead of fully erroring I would just adjust the embed field to be "Unknown" or something. That way, even though the bloodmoon data is missing, the users still get the other info from the command

Reply:
Alright, sounds good. I'll see if I can figure a way to make it unable to retrieve the data to test the case